core/app/components/pubs/PubEditor/PubEditor.tsx

```diff
@@ -62,7 +78,15 @@ export async function PubEditor(props: PubEditorProps) {
 	}
 
 	const [pubs, pubTypes] = await Promise.all([
-		getPubs({ communityId: community.id }),
+		getPubsWithRelatedValuesAndChildren(
+			{ communityId: community.id },
+			{
+				withLegacyAssignee: true,
+				withPubType: true,
+				withStage: true,
+				limit: 30,
```

Comment:
extremely arbitrary number, in the future this will be an api call any so it's fine